pyproject.toml

```diff
@@ -35,7 +35,7 @@ license = { text = "BSD-3-Clause" }
 requires-python = ">=3.6"
 dependencies = [
     "numpy",
-    "pynvml",
+    "pynvml >=11.4.1",
```

Comment:
Why this change?

Reply:
Maybe this comment ( https://github.com/rapidsai/ucx-py/pull/930#discussion_r1115918109 ) helps explain it?

Reply:
Aha, thanks.